multicluster/charts/linkerd-multicluster/templates/controller/deployment.yaml

```diff
@@ -0,0 +1,105 @@
+{{- range .Values.controllers }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    linkerd.io/extension: multicluster
+    component: controller
+    mirror.linkerd.io/cluster-name: {{.link.ref.name}}
+    {{- with $.Values.commonLabels }}{{ toYaml . | trim | nindent 4 }}{{- end }}
```

Comment:
for my education, what is the difference between `.Values` and `$.Values`?

Reply:
The template is wrapped with `{{- range .Values.controllers }}`, so the dot refers to each controller entry and we're forced to use `$.` to refer to the root scope.